tests/components/light/test_init.py

```diff
@@ -313,9 +332,10 @@ def test_broken_light_profiles(self):
             user_file.write("id,x,y,brightness\n")
             user_file.write("I,WILL,NOT,WORK\n")
 
-        assert not setup_component(
-            self.hass, light.DOMAIN, {light.DOMAIN: {CONF_PLATFORM: "test"}}
-        )
+        assert not setup_component(self.hass, light.DOMAIN,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)